cli/lib/util/env.ts

```diff
@@ -25,31 +24,37 @@ export const checkIsRoot = () => {
   }
 };
 
-export const getEnvConfig = () => {
-  const { parsed } = dotenv.config({ path: `${process.cwd()}/.env` });
+export const isCwdInRoot = (params?: {currentPath?: string; alert?: boolean}) => {
+  const currentDir = params?.currentPath || process.cwd();
+  let isInRoot = true;
+  if (!fs.existsSync(join(currentDir, 'package.json'))) {
+    isInRoot = false;
+  } else {
+    const pkg = fs.readFileSync(join(currentDir, 'package.json'), 'utf8');
+    const { name } = JSON.parse(pkg);
+    if (name !== 'erda-ui') {
+      isInRoot = false;
```

Comment:
It's better to prompt user the directory name must be `erda-ui`.

Reply:
this is not verifying directory name, it just checks whether package.json project name is erda-ui